pimegaApp/Db/pimega540d.substitutions

```diff
@@ -1,3 +1,7 @@
+file "pimega.template" {
+  pattern {} {}
+}
```

Comment:
https://github.com/lnls-dig/afc-epics-ioc/blob/master/utcaApp/Db/acq.substitutions#L3-L5

Omitting `pattern {}` works as well, and I think it's slightly clearer.

Reply:
I simply remember it not working for me, because I interpreted it in a similar way.

Reply:
Opened https://github.com/epics-base/epics-base/issues/666 to track this.